### DISCUSSION TITLE
Does 0.1.1 create Self Pool NFT compatible plots?

### DISCUSSION THREAD

**Discussion by GBDownUnder** [madMAx43v3r/chia-plotter#761] [General]
Can you create new Self pool plots with 0.1.1 and use them for other pools later?

I am thinking of the most generic way to create a plot but still be pool compatible.

Which flags would be required for this?  Is the Farmer Public Key the same as for OG plots?
  -p, --poolkey arg    Pool Public Key (48 bytes)
  -c, --contract arg   Pool Contract Address (64 chars)
  -f, --farmerkey arg  Farmer Public Key (48 bytes)

I want to start plotting but haven't decided on a pool yet.

Cheers

**Comment by SebMoore** [comment]
Yep! You can - you don't need the poolkey arg, just the contract arg and farmerkey args. Then later, you can just move to a pool.

**Reply by GBDownUnder** [reply]
What would you use for contract arg if you don't want to specify a particular pool?  Just the same as your farmerkey?

**Reply by SebMoore** [reply]
You need to put a contract arg. To generate one, make a plot NFT: instructions here https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide. If you don't want to join a pool yet, just use the "self pool" option when making the NFT. You can always move to a pool later without replotting if you choose this option.

**Reply by GBDownUnder** [reply]
Thanks man, I am (probably like many) trying to get my head around what this contract key is, is it unique to me, a pool, or me and a pool (as in the combination).  Also what NFT means, etc

Typical learning curve :)

**Reply by SebMoore** [reply]
No worries - totally get it! It's all very confusing haha

**Reply by automation13** [reply]
Guys, I'm confused. Can I do solo farming as before 1.2.0? Just like create plot and farm. Or I need to create Self-Pool and put my plots there?

**Reply by SebMoore** [reply]
You can continue to make and use solo plots just like before! There is no need to self-pool unless you want to.

**Reply by GBDownUnder** [reply]
Yeah I get that, I just wanted to start 'converting' my plots to pool compatible plots before I selected a pool.  In the end I just picked one and started to use its contract address as the doco says I can change pools later anyway so I shouldn't be stuck with my first choice :)

**Reply by SebMoore** [reply]
Yeah exactly - that's what a lot of people are doing at the moment.

**Reply by altendky** [reply]
The contract address isn't the pool's.  It is your NFT's.  So your plots would be tied to your NFT.  You can't change this.  Your NFT can reference a pool, or not.  This is a bit that you can change.  You can join a pool, switch pools, or leave.

Follow the instructions in the above linked wiki page.  You update to 1.2.0, sync, create an NFT (immediately for a pool or not), then start plotting for that NFT's pool contract address.
